### DISCUSSION TITLE
Ccache for android react-native project

### DISCUSSION THREAD

**Discussion by shaylagrymaloski** [ccache/ccache#1392] [General]
Reading this react native doc it seemed like adding ccache to an android project would significantly improve the build speeds https://reactnative.dev/docs/build-speed. When configuring it myself, the project didn't show any improvement. This is what showed in ccache -s when I ran the project twice. Did anyone else experience this?
<img width="668" alt="Screenshot 2024-02-02 at 9 39 15 AM" src="https://github.com/ccache/ccache/assets/41487690/5ef71c1b-8a0e-4a6e-9320-bb81fade0432">

**Comment by fabrizioduronilastminute** [comment]
I was trying ccache today, and on android ssems like it is not called. Did you find any solution/the root cause for this issue?
